controllers/backplaneconfig_controller.go

```diff
@@ -312,6 +312,14 @@ func (r *MultiClusterEngineReconciler) SetupWithManager(mgr ctrl.Manager) error
 				}
 			},
 		}, builder.WithPredicates(predicate.LabelChangedPredicate{})).
+		Watches(&source.Kind{Type: &configv1.ClusterVersion{}}, &handler.Funcs{
```

Comment:
This watch will be impacted by the same bug in https://github.com/stolostron/backlog/issues/26671